src/app/explorePools/page.tsx

```
@@ -558,22 +589,67 @@ function ExploreFatePoolsClient() {
+  // Clicking a column header re-sorts by that field; clicking the active field flips order.
+  const handleSort = useCallback((field: PoolSortField): void => {
+    setSortState(prev =>
+      prev.field === field
+        ? { field, order: prev.order === 'asc' ? 'desc' : 'asc' }
+        : { field, order: field === 'name' ? 'asc' : 'desc' }
+    );
+  }, []);
+
+  // Facet options derived from the loaded pools (distinct base tokens + price feeds).
+  const baseTokenOptions = useMemo((): string[] =>
+    Array.from(new Set(pools.map(p => p.baseSymbol).filter(s => s && s !== "UNKNOWN"))).sort(),
```

ESLint

[error] Use spread syntax [...iterable] instead of Array.from(iterable) when no mapper function is provided

(e18e/prefer-spread-syntax)